src/sync/result-set.js

```diff
@@ -28,7 +28,9 @@ const syncResultSet = async ({ startBlock, endBlock, syncPromises, limit }) => {
           // Parse and insert result set
           const resultSet = parseResultSet({ log: logObj });
           await DBHelper.insertResultSet(resultSet);
-
+          if (resultSet.eventRound === 0) {
```

Comment:
no need to check, this event is always round 0 result sets. `vote-result-set` is the ones that are voted over the threshold.